File: Week1/1-10_최윤성_20210705.py
testcase = int(input())
name = []

for i in range(testcase):
    name.append(input()[0])

result = [i for i in set(name) if name.count(i) >= 5]
result.sort()
        
if not result:
    print("PREDAJA")
else:
    print("".join(result))

File: Week1/1-10_최윤성_20210705.cpp
#include <iostream>
#include <unordered_map>
#include <algorithm>

using namespace std;

int main() {
    unordered_map<char, int> count;
    bool surrender = true;
    string name;
    int testcase;

    cin >> testcase;
    for(int i = 0; i < testcase; i++) {
        cin >> name;
        count[name[0]]++;
    }

    name = "";
    for(unordered_map<char, int>::iterator iter = count.begin(); iter != count.end(); iter++)
        if(iter->second >= 5) {
            name += iter->first;
            surrender = false;
        }
    sort(name.begin(), name.end());
    cout << (surrender?"PREDAJA":name) << endl;
}